<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MIXconfig</class>
 <widget class="QDialog" name="MIXconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">WILMix details</string>
  </property>
  <widget class="QDialogButtonBox" name="closeButtons">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="frame_proxy">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Proxy Process Data</string>
   </property>
   <widget class="QSpinBox" name="proxy_recvPort">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>20</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>data sent to this port is forwarded to the SMi's /process/</string>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="proxy_sendHost">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>process-data gets forwarded to this host</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="proxy_sendPort">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>process-data gets forwarded to this port</string>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_sendProcess">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>UDP-server to send process-data to (via OSC)</string>
    </property>
    <property name="text">
     <string>process →</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_recvProcess">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>UDP-port to listen for connections from OSC-client for process-control</string>
    </property>
    <property name="text">
     <string>process↔</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="proxy_recvEnable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="proxy_sendEnable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="frame_sync">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>SyncStreams</string>
   </property>
   <widget class="QPushButton" name="syncButton">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>70</y>
      <width>211</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>in &quot;sync&quot;-mode streaming/recording is sample-synchronous</string>
    </property>
    <property name="text">
     <string>Sync</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_sync">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;sync-state of the &lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;received&lt;/span&gt; streams:&lt;ul&gt;&lt;li&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;freewheeling&lt;/span&gt; (out-of-sync)&lt;/li&gt;&lt;li&gt;current &lt;span style=&quot; font-style:italic;&quot;&gt;receiver-timestamp&lt;/span&gt;&lt;/li&gt;&lt;/ul&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>freewheeling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_syncTS">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="offsetTS">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Timestamp-offset (for RECORDing) in kSamples</string>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_offsetTS">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>102</y>
      <width>52</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">TSoffset</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_TS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TimeStamp:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_TSvalue">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>average time-stamp of all active SMi's</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="frameDebug">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>Debug</string>
   </property>
   <widget class="QComboBox" name="debugLevel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>90</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>control how much debugging output is generated by WILMix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_debugLevel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LogLevel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButtons</sender>
   <signal>accepted()</signal>
   <receiver>MIXconfig</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButtons</sender>
   <signal>rejected()</signal>
   <receiver>MIXconfig</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
